tests/execution_engine/test_sqlalchemy_batch_data.py

```diff
@@ -99,43 +99,23 @@ def test_instantiation_with_and_without_temp_table(sqlite_view_engine, sa):
     )
     assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 3
 
-
-def test_instantiation_with_and_without_temp_table(sqlite_view_engine, sa):
-
-    print(get_sqlite_temp_table_names(sqlite_view_engine))
-    assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 1
-    assert get_sqlite_temp_table_names(sqlite_view_engine) == {"test_temp_view"}
-
-    # When the SqlAlchemyBatchData object is based on a table, a new temp table is NOT created, even if create_temp_table=True
-    SqlAlchemyBatchData(
-        execution_engine=sqlite_view_engine,
-        table_name="test_table",
-        create_temp_table=True,
-    )
-    assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 1
-
-    selectable = sa.select("*").select_from(sa.text("test_table"))
-
-    # If create_temp_table=False, a new temp table should NOT be created
-    SqlAlchemyBatchData(
-        execution_engine=sqlite_view_engine,
-        selectable=selectable,
-        create_temp_table=False,
-    )
-    assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 1
-
-    # If create_temp_table=True, a new temp table should be created
+    # testing whether schema is supported
+    selectable = sa.select("*").select_from(sa.table(name="test_table", schema="main"))
     SqlAlchemyBatchData(
-        execution_engine=sqlite_view_engine,
-        selectable=selectable,
-        create_temp_table=True,
-    )
-    assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 2
-
-    # If create_temp_table=True, a new temp table should be created
-    SqlAlchemyBatchData(
-        execution_engine=sqlite_view_engine,
+        execution_engine=engine,
         selectable=selectable,
         # create_temp_table defaults to True
     )
-    assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 3
+    assert len(get_sqlite_temp_table_names(sqlite_view_engine)) == 4
+
+    # test schema with execution engine
+    # TODO : Will20210222 Add tests for specifying schema with non-sqlite backend that actually supports new schema creation
```

Comment:
after the discussion during standup today, I decided to keep this as a sqlite test, which supports the `main` schema by default.  This does test the change in `sqlalchemy_execution_engine.py` properly, but will never fail since `main` will always exist.  I'll be adding another set of tests that uses the postgres backend as a `ToDo` in Asana.